src/presentational/RadioButtonGroup.jsx

```diff
@@ -0,0 +1,52 @@
+import { Component, PropTypes } from 'react';
+
+export default class RadioButtonGroup extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      checked: null,
+    }
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(e) {
+    this.props.onChange(e);
+    this.setState({checked: e.target.checked});
+  }
```

Comment:
Does `state.checked` ever get used? I can't see it anywhere else in this component.